### PR TITLE
Aden 3782 | add geo key-val to page level params

### PR DESCRIPTION
**Pull request by bognix**

### CODE REVIEW

**Comment by fraszczakszymon** [comment]
Could you merge dev branch?

File: extensions/wikia/AdEngine/js/AdLogicPageParams.js [hunk 1]
```diff
@@ -8,9 +8,10 @@ define('ext.wikia.adEngine.adLogicPageParams', [
 	'wikia.document',
 	'wikia.location',
 	'wikia.window',
+	'wikia.geo',
```
Comment (fraszczakszymon) [inline extensions/wikia/AdEngine/js/AdLogicPageParams.js]: Could you fix alphabetical order?

File: extensions/wikia/AdEngine/js/spec/AdLogicPageParams.spec.js [hunk 2]
```diff
@@ -4,7 +4,12 @@
 describe('AdLogicPageParams', function () {
 	'use strict';
 
-	var logMock = function () { return; };
+	var logMock = function () { return;},
```
Comment (drets) [inline extensions/wikia/AdEngine/js/spec/AdLogicPageParams.spec.js]: Why space character was removed after semicolon? :japanese_ogre:

File: extensions/wikia/AdEngine/js/AdLogicPageParams.js [hunk 3]
```diff
@@ -182,7 +183,8 @@ define('ext.wikia.adEngine.adLogicPageParams', [
 			lang: zoneParams.getLanguage(),
 			wpage: targeting.pageName && targeting.pageName.toLowerCase(),
 			ref: getRefParam(),
-			esrb: targeting.esrbRating
+			esrb: targeting.esrbRating,
+			geo: geo.getCountryCode()
```
Comment (drets) [inline extensions/wikia/AdEngine/js/AdLogicPageParams.js]: Did you consider the case when `getCountryCode` returns `undefined`?

**Comment by drets** [comment]
Minor comments, apart from that - LGTM :+1:

**Comment by drets** [comment]
LGTM :+1:

**Comment by fraszczakszymon** [comment]
LGTM 👍

**Pull request merged**